v5.6.1 (#4141)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.6.1

_Mar 10, 2022_

We'd like to offer a big thanks to the 8 contributors who made this release possible.

> 

Reviewer: I just merged #3848

The highlight is in the PR's description.

### `@mui/x-data-grid@v5.6.1` / `@mui/x-data-grid-pro@v5.6.1`

#### Changes

- [DataGrid] Display column's filter icon if a filter is applied (#4120) @DanailH
- [DataGrid] Do not loop through rows to compute top level rows count wen the tree is flat (#4081) @flaviendelangle
- [DataGrid] Support extending built-in column types (#4114) @cherniavskii

> - [DataGrid] Use `buildWarning` and `wrapWithWarningOnCall` for deprecated methods and wrong usages (#4056) @flaviendelangle

Reviewer 2: ```suggestion
- [core] Use `buildWarning` and `wrapWithWarningOnCall` for deprecated methods and wrong usages (#4056) @flaviendelangle
```

- [DataGridPro] Re-export the components removed by mistake during bundle split (#4134) @flaviendelangle

### Docs

- [docs] Fix links to prevent duplicate search result (#4130) @siriwatknp
- [docs] Fix outdated links to `localeTextConstants.ts` (#4080) @patilvishal755
- [docs] Neglect e2e tests related to search (#4118) @siriwatknp
- [docs] Use regex instead of specific url in e2e-website-tests (#4121) @siriwatknp

### Core

- [core] Enforce `noImplicitAny` (#4084) @cherniavskii

> - [core] Improve the pro support issue template (#4082) @oliviertassinari

Reviewer: ```suggestion
- [core] Improve the Pro support issue template (#4082) @oliviertassinari
```

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.6.1

_Mar 10, 2022_

We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:

- [DataGrid] Allow to add margins or borders between rows (#3848) @m4theushw

  ```tsx
  <DataGrid getRowSpacing={() => ({ top: 10, bottom: 10 })} />
  ```

  Check the [documentation](https://mui.com/components/data-grid/rows/#row-spacing) for more information.

### `@mui/x-data-grid@v5.6.1` / `@mui/x-data-grid-pro@v5.6.1`

#### Changes

> - [DataGrid] Display column's filter icon if a filter is applied (#4120) @DanailH

Reviewer 3: That was my bad.

```suggestion
- [DataGrid] Aways show column's filter icon if a filter is applied (#4120) @DanailH
```
